src/secrets/containers/SecretsIndex.tsx

```diff
@@ -12,20 +12,31 @@ import {pageTitleSuffixer} from 'src/shared/utils/pageTitles'
 
 // Selectors
 import {getOrg} from 'src/organizations/selectors'
+import {AppState, ResourceType} from 'src/types'
+import SecretsTab from 'src/secrets/components/SecretsTab'
+import GetResources from 'src/resources/components/GetResources'
 
-const SecretsIndex: FC = () => {
+type ReduxProps = ConnectedProps<typeof connector>
+
+const SecretsIndex: FC<ReduxProps> = () => {
   const org = useSelector(getOrg)
 
   return (
     <>
       <Page titleTag={pageTitleSuffixer(['Secrets', 'Settings'])}>
         <SettingsHeader />
         <SettingsTabbedPage activeTab="secrets" orgID={org.id}>
-          <div>Placeholder</div>
+          <GetResources resources={[ResourceType.Secrets]}>
+            <SecretsTab />
+          </GetResources>
         </SettingsTabbedPage>
       </Page>
     </>
   )
 }
 
-export default SecretsIndex
+const mstp = (state: AppState) => ({org: getOrg(state)})
+
+const connector = connect(mstp, null)
+
+export default connector(SecretsIndex)
```

Comment:
we're already getting the org in the selector above: 

https://github.com/influxdata/ui/pull/1736/files#diff-60be7299ee28a12b945e5efd3d4e70741fbb0600db9f1c998d0d2d61e3255f42R22

Reply:
Removed.